docs/requirements.txt

```diff
@@ -1,7 +1,7 @@
 
 # Required by the python script for building documentation
-Sphinx==1.8.3
-sphinx-rtd-theme==0.4.2
-sphinx-tabs==1.1.10
+Sphinx>=2,<3
+sphinx-rtd-theme==0.5.0
+sphinx-tabs==1.2.1
 breathe==4.11.1
 exhale==0.2.1
```

Comment:
You can ditch breathe and exhale plugins and their associated setup on conf.py too, they've made it in there from an old C++ project